src/components/AudioPlayer.tsx

```diff
@@ -50,6 +52,17 @@ const AudioPlayer: React.FC<AudioPlayerProps> = ({player, controller}) => {
     setAttribute({loopEnd: value});
   };
 
+  // this should be in the controller
```

Comment:
sorry about the ts ignore. i had do this cause i couldnt figure out the correct type. this really annoyed me